selenium/pom.xml

```diff
@@ -175,7 +175,7 @@
     <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
-      <version>4.7.2</version>
+      <version>4.10.0</version>
```

Comment:
Using localhost caused 403 on > chromedriver 111 https://github.com/SeleniumHQ/selenium/issues/11750#issuecomment-1460580132

Updating to 4.10.0 resolved this - Can move out of this PR if required